src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.OpationStrategy.OperationHandler;
+import core.basesyntax.OpationStrategy.OperationStrategy;
```

Comment:
There seems to be a typo in the package name 'OpationStrategy'. It should likely be 'OperationStrategy'. Please correct the package name to avoid import errors.